kolibri/plugins/coach/assets/src/views/learners/LearnerSummaryPage/index.vue

```diff
@@ -174,7 +170,10 @@
         return this.getLearnersForExam(quiz).includes(this.learner.id);
       },
       quizLink(quizId) {
-        return this.classRoute(PageNames.REPORTS_LEARNER_REPORT_QUIZ_PAGE_ROOT, { quizId });
+        return this.classRoute(PageNames.QUIZ_LEARNER_REPORT, { quizId });
+      },
```

Comment:
I think the route definitions for these page is  missing the `[meta.titleParts. ](https://github.com/learningequality/kolibri/blob/13027-broken-links-learner-summary-page/kolibri/plugins/coach/assets/src/composables/useCoreCoach.js#L46)` which might explain the failed page title. So,  you might  needs to check the route configuration for those specific pages (QUIZ_LEARNER_REPORT and LESSON_LEARNER_REPORT) and ensure their meta fields have titleParts defined.
<img width="603" alt="Screenshot 2025-01-29 at 15 11 10" src="https://github.com/user-attachments/assets/f8dfeed6-c6b3-4903-9938-dd4a3c0ac764" />

Reply:
Looks like it was caused by the appBarTitle being set specifically rather than letting it use the title parts, should be good now.

Thanks @AllanOXDi !